index.html

```diff
@@ -0,0 +1,165 @@
+              <button class="tag"><span class="tag_txt">Javascript</span></button>
+              <button class="tag"><span class="tag_txt">HTML</span></button>
+            </section>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
+
+          </section>
+        </section>
+        <section class="grid_1-item">
+          <!-- <img src="#" alt="" class="img_place_holder"> -->
+          <section class="txt">
+            <p class="project_title">Multi-Post Stories Gain+Glory</p>
+            <section class="tag_ctn">
+              <button class="tag"><span class="tag_txt">Ruby on Rails</span></button>
+              <button class="tag"><span class="tag_txt">CSS</span></button>
+              <button class="tag"><span class="tag_txt">Javascript</span></button>
+              <button class="tag"><span class="tag_txt">HTML</span></button>
+            </section>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
+
+          </section>
+        </section>
+        <section class="grid_1-item">
+          <!-- <img src="#" alt="" class="img_place_holder"> -->
+          <section class="txt">
+            <p class="project_title">Multi-Post Stories Gain+Glory</p>
+            <section class="tag_ctn">
+              <button class="tag"><span class="tag_txt">Ruby on Rails</span></button>
+              <button class="tag"><span class="tag_txt">CSS</span></button>
+              <button class="tag"><span class="tag_txt">Javascript</span></button>
+              <button class="tag"><span class="tag_txt">HTML</span></button>
+            </section>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
+
+          </section>
+        </section>
+        <section class="grid_1-item">
+          <!-- <img src="#" alt="" class="img_place_holder"> -->
+          <section class="txt">
+            <p class="project_title">Multi-Post Stories Gain+Glory</p>
+            <section class="tag_ctn">
+              <button class="tag"><span class="tag_txt">Ruby on Rails</span></button>
+              <button class="tag"><span class="tag_txt">CSS</span></button>
+              <button class="tag"><span class="tag_txt">Javascript</span></button>
+              <button class="tag"><span class="tag_txt">HTML</span></button>
+            </section>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
+
+          </section>
+        </section>
+    </section>
```

Comment:
**OPTIONAL** - You did well going the extra mile to add this section, but this section is not required for this milestone. I advise that you cut this section before merging into the `main` branch when your project is approved, then paste it into the new branch when you start working on the next milestone. Doing this will ensure that the code reviewer can give an extensive review on this section of your project. Please fix this 🙏

Reply:
Ok, I created a backupcode.html file in which I store that part.

Reply:
@mckent05 Thanks for your contributions. Hope I have done all the required modifications. I am now about to request a second review for this part of the project.